config/index.js

```diff
@@ -0,0 +1,50 @@
+/*
+
+	ard-eventhub
+	by SWR audio lab
+
+*/
+
+// import version from package.json
+const { version } = require('../package.json')
+
+// read env vars
+const stage = process.env.STAGE
+
+// set config
+const serviceName = 'ard-eventhub'
+const baseConfig = {
+	userAgent: `${serviceName}/${version}`,
+	pubsubPrefix: 'de.ard.eventhub',
+	stage,
+	version,
+}
+
+// set config based on stages
+const config = {
+	dev: {
+		...baseConfig,
+		serviceName: `${serviceName}-dev`,
+	},
+	prod: {
+		...baseConfig,
+		serviceName,
+	},
+}
+
+// check stage and config
+if (!stage || !config[stage]) {
+	console.error('STAGE not found >', stage)
+	process.exit(1)
+}
+
+// check existence of several process vars
+if (!process.env.GCP_PROJECT_ID) {
```

Comment:
We could move this block up to line 10, since if these vars don't exist we could exit early.